File: Sources/FocusCore/FocusEntry.swift
import Foundation

public enum FocusEntry: String, Codable, CaseIterable, Sendable {
    case dynamic
    case search

    public var title: String {
        switch self {
        case .dynamic:
            return "动态"
        case .search:
            return "搜索"
        }
    }
}

File: focus-core/src/commonMain/kotlin/org/bilibilifocus/core/model/FocusEntry.kt
package org.bilibilifocus.core.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
enum class FocusEntry {
    @SerialName("dynamic") DYNAMIC,
    @SerialName("search") SEARCH;

    val title: String
        get() = when (this) {
            DYNAMIC -> "动态"
            SEARCH -> "搜索"
        }
}
